utils.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+"""
+Helper functions
+"""
+
+
+def is_service_active(ssh, name):
+    """Helper function for check_services_running to check if a specific service is running"""
+    stdin, stdout, stderr = ssh.exec_command("systemctl status {}".format(name))
+    status = stdout.read()
+    return "Active: active (running)" in status
```

Comment:
Some services are 'Active: active(exited)` so I was thinking can we pass additional parameter for the string to be checked in status? Let me know if it makes sense. 
```
systemctl status target
● target.service - Restore LIO kernel target configuration
   Loaded: loaded (/usr/lib/systemd/system/target.service; enabled; vendor preset: disabled)
   Active: active (exited) since Mon 2019-05-06 11:33:09 EDT; 1 months 21 days ago
```

Reply:
same as above